cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -124,9 +124,10 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Run vulnerability d
 
         val ortResult = advisor.retrieveVulnerabilityInformation(input, skipExcluded).mergeLabels(labels)
 
+        outputDir.safeMkdirs()
```

Comment:
Did this actually fix a bug?

Reply:
Yes. Well, I didn't verify it yet, but we *did* run into an

```
Writing scan result to '/output/results/scan-result.yml'.
Exception in thread "main" java.io.FileNotFoundException: /output/results/scan-result.yml (No such file or directory)
```

error.

Reply:
Thanks!